worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/regions/plotsquared/FaweDelegateSchematicHandler.java

```diff
@@ -65,6 +69,19 @@ public void paste(
             final int zOffset,
             final boolean autoHeight,
             final RunnableVal<Boolean> whenDone
+    ) {
+        paste(schematic, plot , xOffset, yOffset, zOffset, autoHeight, null, whenDone);
+    }
+
+    public void paste(
```

Comment:
Is that API and needed for P2? Otherwise we could make it private.

Reply:
P2 will use the undeprecated method when the PR is merged, then we can delete the method that delegates with a null actor (or leave it)